### PR TITLE
Configurable web interface address

### PR DESCRIPTION
**Pull request by jamesanto**
With this MR, we can configure the web host(0.0.0.0) and port(8080). Since many java based web applications run at 8080 by default, it would be better if we can run the visualization at a different port.

### CODE REVIEW

**Comment by ouven** [comment]
Hi, sorry for responding so late.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Created by .ignore support plugin (hsz.mobi)
```
Comment (ouven) [inline .gitignore]: I prefer to have no .gitignore file in my projects. I rather add a .gitignore with ".gitignore" in the first line.

File: .doctmpl/README.md [hunk 2]
```diff
@@ -43,7 +43,18 @@ de.aktey.akka.visualmailbox {
 }
 ```
 
-It servers a web surface at [http://localhost:8080](http://localhost:8080), that renders the messages with vis.js.
+It servers a web surface at [http://localhost:8080](http://localhost:8080), that renders the messages with vis.js
```
Comment (ouven) [inline .doctmpl/README.md]: This file is generated. Please put your changes into ./doctmpl/README.md

File: .gitignore (hunk 1 above)
Comment (jamesanto) [inline .gitignore]: Removed it.

File: .doctmpl/README.md (hunk 2 above)
Comment (ouven) [inline .doctmpl/README.md]: Oh sorry, did not see. Not enough coffee yet :)

Comment (jamesanto) [inline .doctmpl/README.md]: Did you mean ```./README.md```?

**Pull request merged**